advanced/is-pattern-2/README.md

```diff
@@ -0,0 +1,281 @@
+
+For advanced details on the deployment pattern, please refer the official
+[documentation](https://is.docs.wso2.com/en/latest/setup/deployment-guide/#deployment-patterns).
+
+## Contents
+
+* [Prerequisites](#prerequisites)
+* [Quick Start Guide](#quick-start-guide)
+* [Configuration](#configuration)
+* [Runtime Artifact Persistence and Sharing](#runtime-artifact-persistence-and-sharing)
+* [Managing Java Keystores and Truststores](#managing-java-keystores-and-truststores)
+* [Centralized Logging](#centralized-logging)
+
+## Prerequisites
+
+* WSO2 product Docker images used for the Kubernetes deployment.
+  
+  WSO2 product Docker images available at [DockerHub](https://hub.docker.com/u/wso2/) package General Availability (GA)
+  versions of WSO2 products with no [WSO2 Updates](https://wso2.com/updates).
+
+  For a production grade deployment of the desired WSO2 product-version, it is highly recommended to use the relevant
+  Docker image which packages WSO2 Updates, available at [WSO2 Private Docker Registry](https://docker.wso2.com/). In order
+  to use these images, you need an active [WSO2 Subscription](https://wso2.com/subscription).
+  <br><br>
+
+* Install [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git), [Helm 3](https://helm.sh/docs/intro/install/)
+  and [Kubernetes client](https://kubernetes.io/docs/tasks/tools/install-kubectl/) in order to run the steps provided in the
+  following quick start guide.<br><br>
+
+* An already setup [Kubernetes cluster](https://kubernetes.io/docs/setup).<br><br>
+
+* Install [NGINX Ingress Controller](https://kubernetes.github.io/ingress-nginx/deploy/).<br><br>
+
+* Add the WSO2 Helm chart repository.
+
+    ```
+     helm repo add wso2 https://helm.wso2.com && helm repo update
+    ```
+
+## Quick Start Guide
+
+### 1. Install the Helm Chart
+
+You can install the relevant Helm chart either from [WSO2 Helm Chart Repository](https://hub.helm.sh/charts/wso2) or by source.
+
+**Note:**
+
+* `NAMESPACE` should be the Kubernetes Namespace in which the resources are deployed.
+
+#### Install Chart From Source
+
+>In the context of this document, <br>
+>* `KUBERNETES_HOME` will refer to a local copy of the [`wso2/kubernetes-is`](https://github.com/wso2/kubernetes-is/)
+Git repository. <br>
+>* `HELM_HOME` will refer to `<KUBERNETES_HOME>/advanced`. <br>
+
+##### Clone the Helm Resources for WSO2 Identity Server Git repository.
+
+```
+git clone https://github.com/wso2/kubernetes-is.git
+```
+
+##### Deploy Helm chart for a clustered deployment of WSO2 Identity Server.
+
+ **Helm version 3**
+
+ - Create the Kubernetes Namespace to which you desire to deploy the Kubernetes resources.
+ 
+    ```
+    kubectl create ns <NAMESPACE>
+    ```
+
+ - Deploy the Kubernetes resources using the Helm Chart
+ 
+    ```
+    helm install <RELEASE_NAME> <HELM_HOME>/is-pattern-1 --version 5.10.0-2 --namespace <NAMESPACE> --dependency-update
```

Comment:
@darsham3 please update the release version to `5.10.0-3` as this will be the next release number. Please fix applicable, similar scenarios.